### ISSUE TITLE
Enable learning rate scheduling

### ISSUE THREAD

**Issue by nrweir** [CosmiQ/solaris#27]
@nrweir commented on [Wed Apr 17 2019](https://github.com/CosmiQ/cw-nets/issues/52)

Enable learning rate scheduling for:
- [ ] tensorflow/keras
- [ ] pytorch
- [ ] darknet (? @avanetten)

**Comment by nrweir** [comment]
Resolved in #73
